teamchains/inputs/invite_obsolete_trick.iced

```diff
@@ -0,0 +1,59 @@
+description: """Admin got tricked into using an old invite. Note that
+this cannot happen anymore, because when replaying team chains,
+invites for users that were added afterwards are automatically cleared
+from ActiveInvites. If SBS message arrived for such invite, admin
+would ignore it because the inviteId would be invalid for them.
+
+But: We want this chain to still be valid, because there may be teams
+in the wild that had similar sigchains, and they shouldn't break. """
+
+users: {
+  "herb": {}
+  "basil": {}
+}
+
+teams: {
+  "cabal": {
+    links: [
+      type: "root"
+      members:
+        owner: ["herb"]
+    ,
+      type: "invite"
+      signer: "herb"
+      invites:
+        writer: [ {
+          id: "54eafff3400b5bcd8b40bff3d225ab27",
+          name: "579651b0d574971040b531b66efbc519%1",
```

Comment:
Comment that this is basil%1.
It could be nice to have the uid calculated but not if it's too much work.